common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Note.scala

```diff
@@ -1,61 +1,61 @@
 package weco.catalogue.internal_model.work
 
-sealed trait Note {
-  val content: String
+case class NoteType(id: String, label: String)
+
+case object NoteType {
```

Comment:
I wonder if it's worth leaving a note here as to why we're changing this to avoid a future person thinking they might make the code neater by using the ADT.

Reply:
We have a few patterns like this used across different places, are they worth documenting globally?